Vet clinic database: query multiple tables

Wahaj-Ali opened the pull request:
### Highlights
![1](https://user-images.githubusercontent.com/111431787/220763625-3de25d7c-d696-4e31-bab1-fe24fc631fc8.jpg)
![2](https://user-images.githubusercontent.com/111431787/220763639-39ef9a4a-a0cd-494d-9f76-417cb20c5f18.jpg)
![3](https://user-images.githubusercontent.com/111431787/220763651-f4bf8058-0a86-4f55-a722-7b4535b89d61.jpg)
![4](https://user-images.githubusercontent.com/111431787/220763660-1ae3b8f5-f305-44c9-b63b-9ee0a2404663.jpg)
![5](https://user-images.githubusercontent.com/111431787/220763673-57b81d66-38a5-4e5c-8369-0586762ecb84.jpg)
![6](https://user-images.githubusercontent.com/111431787/220763690-847773d8-bec4-4a32-b9f8-5f48fc4f0baf.jpg)
![7](https://user-images.githubusercontent.com/111431787/220763702-0a344839-03db-4cf8-a405-a26f914c20e0.jpg)

Wahaj-Ali committed:
query multiple tables

In this commit
- Add quires and other commands to add tables 'owners' and 'species'.
- Add command to modify animals table.

Felix45 commented on schema.sql:
```diff
@@ -10,4 +10,32 @@
- ADD COLUMN species VARCHAR(30);
\ No newline at end of file
+ ADD COLUMN species VARCHAR(30);
+
```
- [ Required ] Please consider adding an `SQL`  query that removes the `species` column from the animals table as specified in the project requirements. Please see the screenshot below and rectify this.

## Project requirements
![Screenshot from 2023-02-23 00-39-13](https://user-images.githubusercontent.com/16392046/220765438-61478389-fd31-40c2-bda9-006ce516d6e3.png)

Felix45 commented on data.sql:
```diff
@@ -21,4 +21,32 @@ VALUES ('Boarmon','2005-06-07',7,TRUE,20.4);
+UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
+UPDATE animals SET owner_id = 2 WHERE name = 'Gabumon' OR name='Pikachu';
+UPDATE animals SET owner_id = 3 WHERE name = 'Devimon' OR name='Plantmon';
+UPDATE animals SET owner_id = 4 WHERE name = 'Charmander' OR name='Squirtle' OR name='Blossom';
```
- [ Required ] Please consider adding an `UPDATE` statement that sets *Dean Winchester* as the owner of `Angemon` and `Boarmon` as specified in the project requirements. Please see the screenshot below and rectify this.

## Project requirements
![Screenshot from 2023-02-23 00-45-07](https://user-images.githubusercontent.com/16392046/220766479-7cf05047-2b8e-4db7-b4c7-1aac6ae02c71.png)

Felix45 commented on queries.sql:
```diff
@@ -72,4 +72,52 @@ GROUP BY species;
+SELECT COUNT(*) as pokemon FROM animals AS a
+JOIN species AS s
+ON a.species_id=s.id
+WHERE s.name='Pokemon';
```
- [ Required ] I think this query would not give the number of animals per species that are there in the database since it only checks for the pokemon species, it would be better if you refactored it as shown below.

```SQL
   SELECT species.name, COUNT(*) FROM animals
   LEFT JOIN species
   ON animals.species_id = species.id
   GROUP BY species.name
```

Felix45 commented on queries.sql:
```diff
@@ -72,4 +72,52 @@ GROUP BY species;
+SELECT COUNT(*) as digimon FROM animals AS a
+JOIN species AS s
+ON a.species_id=s.id
+WHERE s.name='Digimon';
```
- [ Required ] It would be better if you use only one query to see the number of animals per species that are there in the database. If you refactor your code as suggested in the above query, you can then delete this query since the refactored code will give the number of animals for all the available species.

```SQL
   SELECT species.name, COUNT(*) FROM animals
   LEFT JOIN species
   ON animals.species_id = species.id
   GROUP BY species.name
```

Felix45 reviewed (changes requested):
Hi @Wahaj-Ali , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://camo.githubusercontent.com/56bcd754643682fcc078b66eaa7278dead6c9c5013595c875b4ddcbe0f0ef0aa/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f78554f786668365a4d373565664d3342716f2f67697068792d646f776e73697a65642d6d656469756d2e676966" />

## Highlights :heavy_check_mark: 
- [x] Gitflow is used correctly :white_check_mark: 
- [x] Repository has a professional readme file :white_check_mark:  


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Wahaj-Ali committed:
Make Changes

In this commit
- Update query to find number of animals per species.
- Add command to make Dean Winchester own Angemon and Boarmon.

Wahaj-Ali committed:
Update schema.sql

- Add command to remove species column

Mihndim2020 reviewed (approved):
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Wahaj-Ali, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Clean and Organized code

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

![Congratulations-Memes-FEATURED](https://user-images.githubusercontent.com/65852621/133696300-b506e476-dc1a-443e-935a-c6f434e788b6.jpg)

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
